### PR TITLE
Add contribution guide.

### PR DESCRIPTION
**Pull request by kyrofa**
This PR adds contribution guidelines that will be picked up by GitHub and suggested as reading material whenever a developer tries to submit a PR.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+   feature on which you'd like to work. Make your changes there, adding new
+   tests as needed, and make sure the existing tests continue to pass when your
+   changes are complete (for information about running the tests, see the
+   [HACKING][1] document).
+
+2. Squash commits into one, well-formatted commit.
+
+   If you really feel like there should be more than one commit in your branch,
+   then you're probably trying to introduce more than one feature and you should
+   make another branch for it.
```
Comment (elopio) [inline CONTRIBUTING.md]: Ok, will comply :)

**Comment by elopio** [comment]
lgtm. I think this document should mention that we need the contributors to sign the license agreement, but that can be a separate PR.

Thanks!

**Comment by kyrofa** [comment]
@elopio ah, I forgot about that, you're right. I don't remember where to point them though... ?

**Comment by elopio** [comment]
@kyrofa: http://www.ubuntu.com/legal/contributors/

**Comment by kyrofa** [comment]
@elopio thanks, I added that in. Good catch!

**Comment by elopio** [comment]
Good, lets merge.

**Pull request merged**